### DISCUSSION TITLE
Is it possible to distinguish between entity types?

### DISCUSSION THREAD

**Discussion by redstar12** [explosion/spaCy#13260] [Help: Coding & Implementations]
Is it possible to distinguish between entity types? e.g. whether the recognized ORG is a company, agency or institution or whether the recognized FAC is an airport highway, bridge etc.?

**Comment by svlandeg** [comment]
We typically advice to label entities with broad categories, as Named Entity Recognition won't necessarily see the difference in text between different types of organizations (if there are no lexical clues). To do this, you start getting more into the realm of [Entity linking](https://spacy.io/api/entitylinker) / disambiguation. That said - you could always try training a [spancat](https://spacy.io/api/spancategorizer) or [ner](https://spacy.io/api/entityrecognizer) model on these type of sublabels. For instance, bridges vs highways might be doable to recognize as I'd assume the lexical context will provide this information.
